builtin/providers/aws/resource_aws_dynamodb_table.go

```diff
@@ -283,6 +283,7 @@ func resourceAwsDynamoDbTableCreate(d *schema.ResourceData, meta interface{}) er
 		} else {
 			// No error, set ID and return
 			d.SetId(*output.TableDescription.TableName)
+			d.Set("arn", *output.TableDescription.TableARN)
```

Comment:
I'm curious, does this work without specifying `arn` in the schema? I think `arn` needs to be defined above, like the [Kinesis resource][1] (though you can omit the `optional` and just leave it as computed). 

[1]: https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_kinesis_stream.go#L33